package.json

```diff
@@ -116,10 +116,11 @@
     "tmp": "0.0.33",
     "uuid": "^9.0.0",
     "vscode-html-languageservice": "^5.0.1",
-    "vscode-jsonrpc": "8.2.0-next.0",
-    "vscode-languageclient": "8.2.0-next.1",
-    "vscode-languageserver-protocol": "3.17.4-next.1",
-    "vscode-languageserver-textdocument": "^1.0.5",
+    "vscode-jsonrpc": "9.0.0-next.7",
```

Comment:
I think the new server package version is available now, we should probably include it in this PR

Reply:
I also need to update the diagnostic integration tests for this change.